app/views/taxons/index.html.erb

```diff
@@ -1,11 +1,14 @@
 <%= display_header title: t("views.taxons.#{params[:action]}.title"), breadcrumbs: ['Taxons'] do %>
+  <%= link_to taxon_path(GovukTaxonomy::ROOT_CONTENT_ID), class: 'btn btn-md btn-default' do %>
+    View homepage taxon
```

Comment:
My thinking is that the homepage is the homepage, and we treat it as a taxon. This could be mistaken for there being a homepage, and then a homepage taxon.

Anyway, not something to worry about too much, but hopefully over time the naming will sort itself out.

Reply:
Interesting point, I didn't think of that. Perhaps we could have used the taxon's internal/external name instead (GOV.UK homepage) as well.

However, we probably want to be consistent with what this taxon is called, because now we have:

- homepage (as introduced in this PR)
- GOV.UK homepage (taxon internal & external name)
- Root of the taxonomy (the taxon description at the top of the visualisations).

I also thinking that we could be continuing with the "Topic Taxonomy" name dubbed in Q3, but that seems to have faded away..